plugins/modules/postgresql_owner.py

```diff
@@ -119,13 +119,13 @@
     obj_name: ssd
     obj_type: tablespace
 
-- name: Reassign all object in database bar owned by bob to alice
+- name: Reassign all databases owned by bob to alice and all objects in database bar owned by bob to alice
```

Comment:
```suggestion
- name: Reassign all databases and all objects in them owned by bob to alice
```

Reply:
This is incorrect as suggested:
```
(postgres@192.168.7.7:5432/postgres[1973715]) # CREATE USER bob;
CREATE ROLE
Time: 32.029 ms
(postgres@192.168.7.7:5432/postgres[1973715]) # CREATE USER alice;
CREATE ROLE
Time: 32.121 ms
(postgres@192.168.7.7:5432/postgres[1973715]) # CREATE DATABASE bob OWNER bob;
CREATE DATABASE
Time: 1274.164 ms (00:01.274)
(postgres@192.168.7.7:5432/postgres[1973715]) # CREATE DATABASE alice OWNER bob;
CREATE DATABASE
Time: 1351.068 ms (00:01.351)
(postgres@192.168.7.7:5432/postgres[1973715]) # \c bob bob
You are now connected to database "bob" as user "bob".
(bob@192.168.7.7:5432/bob[1973715]) > create table bob_in_bob();
CREATE TABLE
Time: 7.333 ms
(bob@192.168.7.7:5432/bob[1973715]) > \c alice bob
You are now connected to database "alice" as user "bob".
(bob@192.168.7.7:5432/alice[1973715]) > create table bob_in_alice();
CREATE TABLE
Time: 21.141 ms
(bob@192.168.7.7:5432/alice[1973715]) > \c postgres postgres
You are now connected to database "postgres" as user "postgres".
(postgres@192.168.7.7:5432/postgres[1973715]) # \l
                                   List of databases
┌───────────┬──────────┬──────────┬─────────────┬─────────────┬───────────────────────┐
│   Name    │  Owner   │ Encoding │   Collate   │    Ctype    │   Access privileges   │
├───────────┼──────────┼──────────┼─────────────┼─────────────┼───────────────────────┤
│ alice     │ bob      │ UTF8     │ en_US.UTF-8 │ en_US.UTF-8 │                       │
│ bob       │ bob      │ UTF8     │ en_US.UTF-8 │ en_US.UTF-8 │                       │
│ doug      │ doug     │ UTF8     │ en_US.UTF-8 │ en_US.UTF-8 │                       │
│ postgres  │ postgres │ UTF8     │ en_US.UTF-8 │ en_US.UTF-8 │                       │
│ template0 │ postgres │ UTF8     │ en_US.UTF-8 │ en_US.UTF-8 │ =c/postgres          ↵│
│           │          │          │             │             │ postgres=CTc/postgres │
│ template1 │ postgres │ UTF8     │ en_US.UTF-8 │ en_US.UTF-8 │ =c/postgres          ↵│
│           │          │          │             │             │ postgres=CTc/postgres │
└───────────┴──────────┴──────────┴─────────────┴─────────────┴───────────────────────┘
(6 rows)

(postgres@192.168.7.7:5432/postgres[1973715]) # REASSIGN OWNED BY bob TO alice;
REASSIGN OWNED
(postgres@192.168.7.7:5432/postgres[1973715]) # \l
                                   List of databases
┌───────────┬──────────┬──────────┬─────────────┬─────────────┬───────────────────────┐
│   Name    │  Owner   │ Encoding │   Collate   │    Ctype    │   Access privileges   │
├───────────┼──────────┼──────────┼─────────────┼─────────────┼───────────────────────┤
│ alice     │ alice    │ UTF8     │ en_US.UTF-8 │ en_US.UTF-8 │                       │
│ bob       │ alice    │ UTF8     │ en_US.UTF-8 │ en_US.UTF-8 │                       │
│ doug      │ doug     │ UTF8     │ en_US.UTF-8 │ en_US.UTF-8 │                       │
│ postgres  │ postgres │ UTF8     │ en_US.UTF-8 │ en_US.UTF-8 │                       │
│ template0 │ postgres │ UTF8     │ en_US.UTF-8 │ en_US.UTF-8 │ =c/postgres          ↵│
│           │          │          │             │             │ postgres=CTc/postgres │
│ template1 │ postgres │ UTF8     │ en_US.UTF-8 │ en_US.UTF-8 │ =c/postgres          ↵│
│           │          │          │             │             │ postgres=CTc/postgres │
└───────────┴──────────┴──────────┴─────────────┴─────────────┴───────────────────────┘
(6 rows)

(bob@192.168.7.7:5432/alice[1973715]) > \c bob postgres
You are now connected to database "bob" as user "postgres".
(postgres@192.168.7.7:5432/bob[1973715]) # REASSIGN OWNED BY bob TO alice ;
REASSIGN OWNED
(postgres@192.168.7.7:5432/bob[1973715]) # \d
           List of relations
┌────────┬────────────┬───────┬───────┐
│ Schema │    Name    │ Type  │ Owner │
├────────┼────────────┼───────┼───────┤
│ public │ bob_in_bob │ table │ alice │
└────────┴────────────┴───────┴───────┘
(1 row)

(postgres@192.168.7.7:5432/bob[1973715]) # \c alice bob
You are now connected to database "alice" as user "bob".
(bob@192.168.7.7:5432/alice[1973715]) > \d
            List of relations
┌────────┬──────────────┬───────┬───────┐
│ Schema │     Name     │ Type  │ Owner │
├────────┼──────────────┼───────┼───────┤
│ public │ bob_in_alice │ table │ bob   │
└────────┴──────────────┴───────┴───────┘
(1 row)
```

Reply:
`REASSIGN OWNED is often used to prepare for the removal of one or more roles. Because REASSIGN OWNED does not affect objects within other databases, it is usually necessary to execute this command in each database that contains objects owned by a role that is to be removed.` hmm